### ISSUE TITLE
Add metrics

### ISSUE THREAD

**Issue by piotlinski** [piotlinski/pytorch-ssd#3]
Now, the only way to measure the training process is the loss function. We could use some metrics to evaluate the model

Criteria:
* [x] search for appropriate metrics and their implementations (i.e. mAP)
* [x] add metrics calculation to training and evaluation
